src/assignments/assignments.yaml

```diff
@@ -22,6 +22,20 @@
 - yugipedia: 371832
   fake_password: 10000090
 ## Prereleases
+# Deck Build Pack: Wild Survivors
+- set_abbreviation: DBWS
+  fake_password_range:
+    - 100430000
+# Duelist Nexus
+- set_abbreviation: DUNE
```

Comment:
To keep track of, looks like this isn't confirmed so the cards won't have the placeholder IDs assigned just yet: https://yugipedia.com/wiki/Duelist_Nexus